lib/stitch_fix/log_weasel/middleware.rb

```diff
@@ -10,18 +14,19 @@ def initialize(app, options = {})
     # Future: Maybe add X-Amzn-Trace-Id request header for tracing through load balancer
     # (http://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-request-tracing.html)
     def call(env)
-      x_request_id = env.fetch("HTTP_X_REQUEST_ID", nil)
+      x_correlation_id = env.fetch(REQUEST_ID_KEY, nil) || env.fetch(CORRELATION_ID_KEY, nil)
```

Comment:
This is super not important, but a more "idiomatic ruby" way to do this would be:

```ruby
x_correlation_id = env.fetch(REQUEST_ID_KEY,env.fetch(CORRELATION_ID_KEY,nil))
```

or

```ruby
x_correlation_id = env[REQUEST_ID_KEY] || env[CORRELATION_ID_KEY]
```

Reply:
That first way was how Reed initially had it (except broken out on separate lines), and I suggested the way he has it now which I felt was more idiomatic. ;-) I didn't love the nested `env.fetch`es.

I think you can leave it alone Reed. Sorry for making your life difficult!

Reply:
lol, sorry about that :(. I've never seen `fetch` and `||` used in this way, but you know Ruby…many ways to do it :)

Reply:
My personal idiom is to break cognitive atoms into distinct visual representations. So in this case it was one line break for each potential assignment to the variable. No big deal, leaving as is.

Reply:
http://bikeshed.com :)